scripts/install-go.sh

```diff
@@ -0,0 +1,38 @@
+#!/bin/bash
+
+set -e
+
+export DEBIAN_FRONTEND=noninteractive
+VERSION=1.4.3
+OS=linux
+ARCH=amd64
+
+prefix=/usr/local
+
+which wget || apt-get install -y wget
+# not essential but go get depends on it
+which git || apt-get install -y git
+
+# grab go
+if ! [ -e "$prefix/go" ]; then
+    if ! [ -e "go$VERSION.$OS-$ARCH.tar.gz" ]; then
+        wget -q https://storage.googleapis.com/golang/go$VERSION.$OS-$ARCH.tar.gz
+    fi
+    tar -C $prefix -xzf go$VERSION.$OS-$ARCH.tar.gz
```

Comment:
Those tools are required if you want to run `make check`. So yeah, they are not strictly needed for building, but might be still useful for running tests inside the container.